src/software/test_util/test_util.cpp

```diff
@@ -102,4 +102,50 @@ namespace Test
         return Robot(robot_id_counter++, pt, Vector(), Angle(), AngularVelocity(),
                      Timestamp());
     }
+
+    bool TestUtil::checkGeometryEqualWithTolerance(const Polygon &poly1,
```

Comment:
Why not call this `checkPolygonEqual...`?

Reply:
check___ is actually redundant so changing to equalWithinTolerance